algo-exercise/getting-started/src/test/java/codingblackfemales/gettingstarted/StretchAlgoTest.java

```diff
@@ -138,72 +133,20 @@ protected UnsafeBuffer createTickBUYLow(){
         return directBuffer;
     }
 
-    @BeforeEach
-    public void setUp(){
-        orderBookService = new OrderBookService();
-        marketStatus = new MarketStatus();
-        mwaCalculator = new MovingWeightAverageCalculator(orderBookService);
-        logicInstance = new StretchAlgoLogic(marketStatus, orderBookService, mwaCalculator);
-        MarketStatus marketIsForcedOpen = new MarketStatus() {
-            @Override
-            public boolean isMarketClosed() {
-                return false;  // Force open
-            }
-        };
-        MarketStatus marketIsForcedClosed = new MarketStatus() {
-            @Override
-            public boolean isMarketClosed() {
-                return true;  // Force closed
-            }
-        };
-        marketIsForcedOpenInstance = new StretchAlgoLogic(marketIsForcedOpen, orderBookService, mwaCalculator);
-        marketIsForcedClosedInstance = new StretchAlgoLogic(marketIsForcedClosed, orderBookService, mwaCalculator);
-    }
-
-    // Test 1: Check isMarketClosed behaves as expected
-    @Test
-    public void testIsMarketClosedMethod(){
```

Comment:
this is a bit tricky now if we dont pass the current time to the marketStatus.isMarketClosedMethod(). Ideally current time is passed as an argument. Then we would write a test against sunday/saturday before and after close.

Reply:
I felt change was too big at this point. But something you could mention in your submission.